src/services/auth.js

```diff
@@ -45,3 +45,20 @@ export const postLogoutUser = () => {
     },
   });
 };
+
+export const putUpdateNewPassword = params => {
+  const { temporaryToken, password } = params;
+  const endpoint = `${endpoints.api.reset_password}`;
+  return request.put(endpoint, {
+    data: {
+      // First name is expected
+      // on the server side.
```

Comment:
I'm not sure what this code comment is supposed to be telling me.  Aside from the fact that I believe that the Server expects the _user_ name (not the "First name"), if the server expects it why isn't the call supplying it?

Reply:
It appears that this is being tracked in #211.  Until that's resolved, it would probably be good to reference it in this code comment.